developer_manual/general/codingguidelines.rst

```diff
@@ -311,7 +311,16 @@ Control Structures
 PHP
 ---
 The ownCloud coding style guide is based on `PEAR Coding Standards <http://pear.php.net/manual/en/standards.php>`_.
-To check your PHP codestyle use `PHP Code Sniffer <https://github.com/squizlabs/PHP_CodeSniffer>`_ >= 3.0 with the ``phpcs.xml`` config file from the core branch. e.g. ``phpcs --standard=./phpcs.xml yourCode.php``
+To check your PHP codestyle use `PHP Code Sniffer <https://github.com/squizlabs/PHP_CodeSniffer>`_ >= 3.0 with the ``phpcs.xml`` config file from the core branch.
+
+To check one file use: ``phpcs --standard=./phpcs.xml yourCode.php`` 
+
+To check all files in a folder (recursive) use: ``phpcs --standard=./phpcs.xml your/code/folder/``
+
+A `git pre-commit hook <https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks>`_ is available `here <https://raw.githubusercontent.com/Ikke/git-precommit-phpcs/master/pre-commit>`_. Download and save the file in the ``.git/hooks`` folder of your owncloud project and change the ``PHPCS_STANDARD`` constant to the path of the ``phpcs.xml`` file.
+
+Start & closing
+^^^^^^^^
```

Comment:
@individual-it this looks great. The one, minor, thing that stands out is the length of the header underline on line 323. It seems too short. Can you lengthen it to match the above header title? Other than that, it's 👍 from me. I really appreciate you adding this.